src/modules/tests/test-summary/test-summary.selector.ts

```diff
@@ -18,3 +18,9 @@ export const isDebriefWitnessed = (testSummary: TestSummary): boolean => testSum
 export const isDebriefUnwitnessed = (testSummary: TestSummary): boolean => {
   return (testSummary.debriefWitnessed != null && !testSummary.debriefWitnessed) || false;
 };
+export const debriefWasWitnessed = (testSummary: TestSummary): boolean => {
+  return (testSummary.debriefWitnessed) || false;
+};
+export const debriefWasUnwitnessed = (testSummary: TestSummary): boolean => {
```

Comment:
This is identical to the isDebriefUnwitnessed function above

Reply:
Nice one, Tom. It seems both of these are unused.

Reply:
It looks like these have come in from different versions.  the correct one is isDebriefUnwitnessed.

Reply:
I've removed the offending exports, Ross. Thanks for confirming.